batch.go

```diff
@@ -100,7 +100,7 @@ func (c *Client) ListBatchesWithContext(ctx context.Context, opts *ListOptions)
 // AddShipmentsToBatch adds shipments to an existing batch, and returns the
 // updated batch object.
 func (c *Client) AddShipmentsToBatch(batchID string, in ...*Shipment) (out *Batch, err error) {
-	req := batchRequest{Batch: &Batch{Shipments: in}}
+	req := map[string]interface{}{"shipments": in}
```

Comment:
Is it possible to pass in shipment IDs like our documentation shows?

Reply:
@jchen293 has this been addressed?

Reply:
working on it, doing some testing.

Reply:
After some deliberation, we are going to use `interface{}` so the user can pass in either `Shipment IDs` or `Shipment structs`, this will avoid breaking changes.